### DISCUSSION TITLE
Elephant brain MRI segmentation

### DISCUSSION THREAD

**Discussion by malav4994** [ANTsX/ANTs#1482] [Q&A]
Hi all, I am trying to extract cortical thickness of elephant brain neocortex with structural MRI data. I have figured out a pipeline I can use to generate WM-GM and GM-pial matter surfaces. However, I feel stuck at the segmentation level. As a lot of MRI segmentation tools are optimised for human MRI with probability maps, they are not much helpful with elephant brain segmentation. Can anyone help me with these brains segmentation using other image processing tools?

**Comment by cookpa** [comment]
Can you tell us more about the data? How many images do you have? Can you share an example?

**Reply by malav4994** [reply]
Hi Philip,
I have scans from multiple animals.
Some of them have just one T1 scan (eg a coronal slice from one of the animals)
![image](https://user-images.githubusercontent.com/8775742/211814890-aaaa5264-a82f-428d-80f9-66e86af806dd.png)

For some other animals, we have T1 and T2 scans. (I can not share the data online)
Unfortunately, the data is not great as it has other imaging artefacts due to tissue qualities and practical scanning constraints.
But I would be happy to show in one-to-one interaction, if the project interests you.
Thanks

**Comment by cookpa** [comment]
That does look like unusual contrast. 

I don't have any experience with these images but I can tell you ANTs has the ability to do segmentation without priors using Atropos. You might be able to build a template from multiple images and use segmentation (eg, k-means) either in the template or bootstrapped / averaged from the input images to generate priors. You'll need a brain mask that excludes background material.

If each brain's T1w and T2w are well aligned, the T1w / T2w ratio might be worth computing and looking at, to see if it gives any improvment in contrast.

Cortical thickness measurements can be done if you have a good GM / WM segmentation. I've not tried it on anything but humans but I don't think it has much prior information about the morphology - which might sound good but it also means it's very dependent on the segmentation being correct.

**Reply by malav4994** [reply]
I managed to segment a small volume section with some image processing tools and manual editing. I have these files in the form of nii.gz. How can I generate (white- and pial-matter) surfaces using the segmentation volume, and then estimate cortical thickness?

**Reply by ntustison** [reply]
Just to be clear, ANTs tools operate completely within image volumetric space to compute thickness.  This is completely different from other packages, such as FreeSurfer, which do explicitly construct gray and white matter surfaces for cortical thickness estimation.   I would suggest looking at the results [here](https://github.com/ntustison/antsCorticalThicknessExample), also in .nii.gz format, to understand ANTs output.  If that is adequate for your purposes, we can help you with your data.

**Comment by malav4994** [comment]
Thank you for clarifying, Nick.
I am not looking for a surface-based solution precisely, as long as I can estimate cortical thickness based on the shortest distance between white- and pial-matter within 3 dimensions. I understand ANTs generates the same outcome as I am looking for. So if you can show me how I can move forward with me segmentation volume, that would be really helpful.
Thanks in advance.

**Reply by ntustison** [reply]
If you have the segmentations, then you can use the ``KellyKapowski`` program directly.  Take a look at the help menu and see how it's used in the human ``antsCorticalThickness.sh`` script.  See, for example, [here](https://github.com/ANTsX/ANTs/blob/master/Scripts/antsCorticalThickness.sh#LL403C1-L410C1) and [here](https://github.com/ANTsX/ANTs/blob/master/Scripts/antsCorticalThickness.sh#L1282-L1285).
